plugins/check_invalid_rotation/check_invalid_rotation.js

```diff
@@ -0,0 +1,700 @@
+            fixAllDescription: 'Ceci affecte toutes les rotations qui ont plusieurs options.',
+            fixAllSuccess: 'Toutes les rotations ont été corrigées !',
+            about: `# Vérificateur de Rotations Minecraft
+Détecte les rotations non-standard avant l'export pour éviter les problèmes d'affichage dans Minecraft.
+
+## 🎯 Fonctionnalités
+• **Détection Intelligente** : Identifie les rotations qui ne suivent pas la norme 22,5° de Minecraft Bedrock
+• **Protection d'Export** : Affiche une boîte de dialogue d'avertissement avant l'export des fichiers JSON ou BBModel
+• **Correction Auto** : Corrige automatiquement les rotations vers les valeurs standard les plus proches
+• **Choix de l'Utilisateur** : Continuer l'export ou annuler pour corriger les rotations d'abord
+
+## 📐 Normes Minecraft
+Minecraft Bedrock supporte les rotations par incréments de 22,5° :
+**-45°, -22,5°, 0°, 22,5°, 45°**
+
+Les rotations non-standard peuvent causer des problèmes visuels en jeu.
+
+## 💡 Comment Ça Marche
+1. Le plugin analyse les valeurs de rotation lors de l'export JSON ou BBModel
+2. Affiche une boîte de dialogue d'avertissement si des rotations non-standard sont trouvées
+3. Liste les éléments problématiques avec les corrections suggérées
+4. Choisir de corriger automatiquement ou continuer l'export
+
+*Version 1.0.0 - Compatible avec Blockbench 4.12.6+*`,
+            dialogTitle: 'Rotations Non-Standard Détectées',
+            warningMessage: 'rotation(s) non-standard trouvée(s)',
+            warningDescription: 'Ces rotations peuvent causer des problèmes d\'affichage dans Minecraft. Considérez les corriger avant l\'export.',
+            rotationIssues: 'Problèmes de Rotation :',
+            quickFix: 'Correction Rapide',
+            quickFixDescription: 'Corriger automatiquement toutes les rotations vers leurs valeurs standard les plus proches.',
+            fixAll: 'Tout Corriger',
+            fix: 'Corriger',
+            fixed: 'Corrigé ✓',
+            rotationsFixed: 'rotations corrigées ✓',
+            cancelExport: 'Annuler l\'Export',
+            continueExport: 'Continuer l\'Export',
+        }
+    };
+
+    function detectLanguage() {
+        if (typeof Settings !== 'undefined' && Settings.language) {
+            return Settings.language.startsWith('fr') ? 'fr' : 'en';
+        }
+        
+        const docLang = document.documentElement.lang || navigator.language;
+        if (docLang && docLang.startsWith('fr')) {
+            return 'fr';
+        }
+        
+        return 'en';
+    }
+
+    function t(key) {
+        const lang = detectLanguage();
+        return translations[lang][key] || translations.en[key] || key;
+    }
+
+    Plugin.register('check_invalid_rotation', {
+        title: t('title'),
+        author: 'AnnJ',
+        description: t('description'),
+        icon: 'warning',
+        version: '1.0.0',
+        variant: 'both',
+        tags: ['minecraft', 'rotation', 'validation', 'export'],
```

Comment:
You can specify up to 3 tags. Make sure they are formatted for display in UI, in this case they should start with an upper case letter.